configs/eslint-config-compass/index.js

```diff
@@ -1,143 +1,62 @@
-const path = require('path');
-
-const jsConfigurations = ['eslint:recommended'];
+'use strict';
 
-const tsConfigurations = [
-  ...jsConfigurations,
-  'plugin:@typescript-eslint/recommended',
-  'plugin:@typescript-eslint/recommended-requiring-type-checking',
-];
+const path = require('path');
+const shared = require('@mongodb-js/eslint-config-devtools');
+const common = require('@mongodb-js/eslint-config-devtools/common');
 
 const tsRules = {
-  '@typescript-eslint/no-unused-vars': 'error',
-  '@typescript-eslint/no-unsafe-assignment': 'off',
-  '@typescript-eslint/no-unsafe-call': 'off',
-  '@typescript-eslint/no-unsafe-member-access': 'off',
-  '@typescript-eslint/no-unsafe-return': 'off',
-  '@typescript-eslint/consistent-type-imports': [
-    'error',
-    { prefer: 'type-imports' },
-  ],
+  ...common.tsRules,
```

Comment:
Just a question, is this necessary or `extends: ['@mongodb-js/eslint-config-devtools']` in the override section is enough to cover all of the defaults?

Reply:
It isn't clear to me at all what will happen for arrays of objects where there are no ids and how that will be merged when you specify options for the same thing twice. I feel this removes the ambiguity because at least overrides is only specified once.